### ISSUE TITLE
Isatab isa_16182_919177.zip fails

### ISSUE THREAD

**Issue by flekschas** [parklab/refinery-platform#526]
When importing this isatab file via the command line I get an IO error.

```python manage.py process_isatab admin /vagrant/transfer/path/to/dir/with/isa_16182_919177.zip/```

```IOError(2, 'No such file or directory')```

**Comment by ngehlenborg** [comment]
It works for me through the UI. Can you check?

**Comment by flekschas** [comment]
I get the same error.

**Comment by flekschas** [comment]
I tried it again and I can confirm it fails.

**Comment by ngehlenborg** [comment]
Is your local copy of this archive broken?

**Comment by flekschas** [comment]
Same error with a fresh copy of the file. The file in drive could still be broken though.

**Comment by flekschas** [comment]
Here is the actual error message:

```
IOError: [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/celery/execute/trace.py", line 47, in trace
    return cls(states.SUCCESS, retval=fun(*args, **kwargs))
  File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/celery/app/task/__init__.py", line 247, in __call__
    return self.run(*args, **kwargs)
  File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/celery/app/__init__.py", line 175, in run
    return fun(*args, **kwargs)
  File "/vagrant/refinery/data_set_manager/tasks.py", line 586, in parse_isatab
    fileStoreItem.get_file_object()
  File "/vagrant/refinery/file_store/models.py", line 401, in get_file_object
    return self.datafile.file.file  # FileStoreItem.FieldFile.File.file
  File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/django/db/models/fields/files.py", line 46, in _get_file
    self._file = self.storage.open(self.name, 'rb')
  File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/django/core/files/storage.py", line 36, in open
    return self._open(name, mode)
  File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/django/core/files/storage.py", line 159, in _open
    return File(open(self.path(name), mode))
IOError: [Errno 2] No such file or directory
```

**Comment by flekschas** [comment]
I found the reason.

I successfully imported a isatab into Refinery from my host machine via a synced folder. Then after importing I moved the filed on my host machine somewhere else. The `FileStoreItem` has kept the link to the isatab file though, so when I reimport the same isatab again Refinery can't find the original file anymore since it is still searching for the original location on my host machine. This will consequently through an `IOError`.

**How to reproduce**:
Assume you have `~/data` synced with Refinery. Create a new dir in `~/data` called `test1`, copy an isatab in there and import it from within the vm using `/manage.py process_isatab admin /vagrant/transfer/test1/`. Leave the vm and move `test1` to `test2` and try to re-import. Then you'll see the `IOError`. See the link of `FileStoreItem` in the admin interface. It'll all point to `/vagrant/transfer/test1/...`
